stboot.go

```diff
@@ -31,21 +31,27 @@ import (
 	"github.com/u-root/u-root/pkg/uio"
 )
 
+const (
+	hostCfgDefault = "/etc/host_configuration.json"
+	hostCfgHelp = `Location of Host Configuration file.
+inside initramfs: "/path/to/host_configuration.json"
+STBOOT partition: "legacy:/path/to/host_configuration.json"
```

Comment:
Seems that you do not use `/path/to/host_configuration.json` in that case, but always go with `host.HostConfigFile` (line 185) which is fine from my side, so 
* either delete the `:/path/to/host_configuration.json` here in the description
* or make use of it when set further down

Reply:
indeed, thanks. I'll delete `host.HostConfigFile` and stick with `hstCfg.name`

Reply:
:D that was not my intention.
legacy should make stboot behave like it was: Mout STBOOT and take hostConfig from a fix path.

Reply:
ah ok :sweat_smile:

Reply:
done :+1: